packages/gitbook/src/lib/cache/cache.ts

```diff
@@ -52,6 +52,9 @@ export interface CacheDefinition<Args extends any[], Result> {
     /** Tag to associate to the entry */
     tag?: (...args: Args) => string;
 
+    /** Context key matching the hash of the claims included in the visitor JWT token */
+    getContextKey?: () => string | undefined;
```

Comment:
This comment and even the function name is too specific to a usage that `cache.ts` should not know about.

Reply:
Renamed to `getKeySuffix` but happy to change it if you don't think that works. Other name I was thinking was `getDiscriminatorKey`. Otherwise if you have any other suggestions.